spec/models/folder_spec.rb

```diff
@@ -17,5 +17,10 @@
       expect(subject.unread_count).to eq(params[:unread_count])
       expect(subject.system_folder).to eq(params[:system_folder])
     end
+
+    it 'can be compared by name' do
+      expect(subject <=> other).to eq(-1)
+      expect(other <=> subject).to eq(1)
+    end
```

Comment:
why were any of the changes in this needed? You're not doing anything with folder in this PR as best as I can tell? I had removed the spec for testing comparing because it sporadically ran into issues depending on the order that rspec was run. Does this fix that?